_posts/2013-02-19-background-image-options.md

```diff
@@ -16,15 +16,18 @@ prefixed: false
   /* various options, see e.g. http://www.css3files.com/background */
   background: url(image1.png), 2 url(image2.png), 3 url(image3.png) #FFF;
   background-clip: border-box;
+  /* if value text is used for background-clip, it should be prefixed with webkit */
```

Comment:
Can we explicitly mention that "text" is not-yet-standard?

e.g:

```css
/*
For the not-yet-standard value of ‘text’ WebKit and Edge need `-webkit-`!
Firefox supports 'text' without prefixes:
*/
-webkit-background-clip: text;
background-clip: text;
```

Reply:
Got it!